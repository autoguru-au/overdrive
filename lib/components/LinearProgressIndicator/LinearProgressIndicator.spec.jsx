import React from 'react';
import { render } from '@testing-library/react';
import { LinearProgressIndicator } from '.';

describe('<LinearProgressIndicator />', () => {
	it('should not throw', () =>
		expect(() => render(<LinearProgressIndicator />)).not.toThrow());

	it('should match snapshot', () => {
		expect(
			render(<LinearProgressIndicator />).container.firstChild,
		).toMatchSnapshot();
	});
});
