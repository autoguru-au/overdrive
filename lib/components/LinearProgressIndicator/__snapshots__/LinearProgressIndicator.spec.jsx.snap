// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LinearProgressIndicator /> should match snapshot 1`] = `
<div
  class="root"
>
  <div
    class="linearProgressBar"
  >
    <span
      class="linearProgressBarInner"
    />
  </div>
</div>
`;
